taxonomies/best-practices/name-annotation.md

```diff
@@ -0,0 +1,85 @@
+# Introduction
+
+This document gathers best practices for lightweight ontology and taxonomy design and edition. 
+It should be considered as a collection of guidelines to follow when contributing to Neuroshapes ontologies and taxonomies. 
+Note that below I'll used ontology in general to refer to both ontologies and taxonomies.
```

Comment:
**Maybe**: Below, the term 'ontology' is used to refer to both ontologies as well as taxonomies

Reply:
Yes, better